docs/swagger.yaml

```diff
@@ -188,6 +189,95 @@ paths:
         - $ref: '#/components/parameters/Merchant-Serial-Number'
         - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
         - $ref: '#/components/parameters/Authorization'
+  '/v2/{type}/receipts/{orderId}':
+    parameters:
+      - $ref: '#/components/parameters/Transaction-Type'
+      - $ref: '#/components/parameters/Order-Id'
+    post:
+      summary: Endpoint for adding receipts to an order.
+      operationId: postReceiptV2
+      responses:
+        '201':
+          $ref: '#/components/responses/201Created'
+        '400':
+          $ref: '#/components/responses/400BadRequest'
+        '401':
+          description: Unauthorized
+        '404':
+          $ref: '#/components/responses/404NotFound'
+      tags:
+        - Receipt
+      description: Endpoint for adding receipts to an order.
+      parameters:
+        - $ref: '#/components/parameters/Merchant-Serial-Number'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+        - $ref: '#/components/parameters/Authorization'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Receipt'
+  '/v2/{type}/categories/{orderId}':
+    parameters:
+      - $ref: '#/components/parameters/Transaction-Type'
+      - $ref: '#/components/parameters/Order-Id'
+    put:
+      summary: Post additional information about an order that will show up on the Vipps receipt.
+      operationId: putCategoryV2
+      responses:
+        '200':
```

Comment:
This somehow shows this in swagger? hM
<img width="855" alt="image" src="https://user-images.githubusercontent.com/47554984/160613802-f329673e-95cb-4a39-a555-e760a962b1c2.png">

Reply:
Er du sikker på at du ikke blander med 
`/v2/{paymentType}/receipts/{orderId}`